backend/src/data/migrations/20220716081411_add_users_table.ts

```diff
@@ -3,6 +3,7 @@ import { Knex } from 'knex';
 enum ColumnName {
   ID = 'id',
   EMAIL = 'email',
+  FULL_NAME = 'full_name',
```

Comment:
you can't change old migration. you should create a new one

Reply:
Does it mean, that I should remove the previous one?

Reply:
you should create a new migration where you will add a new field (full_name)
you can't change the old one